frameworks-versions.properties

```diff
@@ -68,7 +68,7 @@ prestoDefaultVersion=0.276
 # Exact Presto versions by Presto major versions
 versionPresto-0.274=0.274
 versionPresto-0.275=0.275
-versionPresto-0.276=0.276
+versionPresto-0.276=0.276.1
```

Comment:
i assume the `0.276` usage further below in this file gets translated the the exact version somewhere?

Reply:
Yep - this one is to resolve a major version to a particular "real" version.